components/crypto/src/lib.rs

```diff
@@ -550,7 +550,11 @@ fn verify_keys_match(public_key: &PublicKey, secret_key: &SecretKey) -> bool {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
```

Comment:
The relevant [clippy docs](https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_imports) state:

> `use super::*` is allowed in test modules. This is defined as any module with “test” in the name.

Is this not the case? IMO, `use super::*` for test modules does make sense.

Reply:
I followed by clippy suggestions only 🙃